images/win/scripts/Installers/Install-Firefox.ps1

```diff
@@ -4,17 +4,63 @@
 ################################################################################
 
 Import-Module -Name ImageHelpers -Force
+Import-Module -Name PathHelpers -Force
 
 $temp_install_dir = 'C:\Windows\Installer'
 New-Item -Path $temp_install_dir -ItemType Directory -Force
 
-Install-EXE -Url "https://seleniumwebdrivers.blob.core.windows.net/knownfirefoxversion/FirefoxSetup.exe" -Name "FirefoxSetup.exe" -ArgumentList "-ms"
+$versionsJson = Invoke-WebRequest  "https://product-details.mozilla.org/1.0/firefox_versions.json" | ConvertFrom-Json
```

Comment:
Replace Invoke-WebRequest to Invoke-RestMethod and  remove | ConvertFrom-Json

Reply:
Done